### DISCUSSION TITLE
Log(HSP v3.7)?

### DISCUSSION THREAD

**Discussion by youxiaojie** [dbuezas/lgt8fx#138] [Q&A]
I looked there is some hsp v3.7 code in original github/lgtmcu, why it is not adopted by this libs?

**Comment by SuperUserNameMan** [comment]
Do you mean here => https://github.com/LGTMCU/Larduino_HSP ??

**Reply by youxiaojie** [reply]
yes

**Reply by SuperUserNameMan** [reply]
it seems outdated. Last update was 2 years ago.

Have you found something new or different ?

**Reply by youxiaojie** [reply]
no,just found these hsp 3.7 code which not shown in this repo. donot why.
it should be a branch of  github/lgtmcu to keep up with upstreams.

**Reply by LaZsolt** [reply]
> just found these hsp 3.7 code which not shown in this repo. donot why.

There is a link on the main page. Just one line upper than Power consumption table.

**Reply by SuperUserNameMan** [reply]
>no,just found these hsp 3.7 code which not shown in this repo. donot why.
it should be a branch of github/lgtmcu to keep up with upstreams.

I don't know why.

But according to [readme](https://github.com/dbuezas/lgt8fx), our lgt8fx came from this [Larduino_HSP_v3.6c](https://github.com/Edragon/LGT/tree/master/1-LGT8F328/SDK/Arduino/HSP/HSP%20Patch%20File/Larduino_HSP_3.6c/Larduino_HSP_v3.6c) and some FastIO code was back ported from the HSP 3.7 you mentioned.

Apparently, @dbuezas discovered LGT8F328p thanks to @RalphBacon who published a video on youtube and who created a repo containing all the information he found, including a [Larduino_HSP_v3.6c.zip](https://github.com/RalphBacon/LGT8F328P-Arduino-Clone-Chip-ATMega328P) file.

So maybe that is why our lgt8fx did not fork from Larduino_HSP. Maybe @dbuezas  started from the .zip file.
